test/e2e/utils/runner.go

```diff
@@ -73,9 +87,78 @@ func (r *Runner) Teardown() {
 		r.containerdSession.Terminate().Wait()
 	}
 
+	cleanupThinPools()
+	cleanupDirectories()
+
 	gexec.CleanupBuildArtifacts()
 }
 
+func makeDirectories() {
```

Comment:
I wonder if we can add this to a `pkg` or somewhere, and reuse in `cmd/dev-helper` and the runner, we could manage this setup from one place.

Reply:
gonna delete `cmd/dev-helper` instead 😈

Reply:
It would be still better to manage those settings in one place, I like the Go version and we can use this as a replacement for the shell script (and update the documentation to use that instead)

Reply:
oh like you mean instead of all the copy-pastable commands we have in the quick-start docs?

for sure, would be good to have a "setup all my stuff" command for people.

can i do that later tho :p